ejenti/jobs/status_json_creator.py

```diff
@@ -127,7 +127,7 @@ def status_json_creator(**kwargs):
                         job_utc_ts = data_obj["start_ts"]
                         task_name = data_obj["cmd"]
                         for status_obj in data_obj["status"]:
-                            status_code = status_obj["code"]
+                            status_code = status_obj["fn_code"]
```

Comment:
what does `fn_code` mean?
what's the difference between `code` and `fn_code`?

Reply:
fn_code means the real status code in file name ex: 900-1-pulse.json, fn_code would be 900-1 and code would be 900

Reply:
👍